image/Dockerfile

```diff
@@ -12,23 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:stretch
+# --------------------------------------------------------------------------------------------------
+# The building stage.
+FROM golang:1.14.2-stretch AS builder
 
-RUN apt-get update
-RUN apt-get install -y curl ca-certificates
+WORKDIR /build/minibroker
+COPY ./go.mod ./go.mod
+COPY ./go.sum ./go.sum
```

Comment:
```suggestion
# Copy the go.mod over so docker can cache the module downloads if possible.
COPY go.mod go.mod ./
```

Reply:
Could you update the suggestion to `go.{mod,sum}`? I don't want to add the same file twice. :sweat_smile:

Reply:
Like this then?
```suggestion
COPY go.mod go.sum ./
```

Reply:
:crossed_fingers: